package.json

```
@@ -14,7 +14,7 @@
   "license": "Apache-2.0",
   "scripts": {
     "build": "bash ./scripts/md2html/build.sh",
-    "build-src": "npm run validate-markdown && bash ./scripts/md2html/build-src.sh",
+    "build-src": "npm run validate-markdown && bash ./scripts/md2html/build.sh oas",
```

```suggestion
    "build-src": "npm run validate-markdown && bash ./scripts/md2html/build.sh src",
```

Eventually we want to reuse this for Arazzo and Overlay, and using the common folder name instead of the spec shortname makes that easier.

---

What would the invocations look like for those other specifications? It's not a difficult change to make, but I guess I don't understand how making it "src" helps us out in the future.

* npm run build arrazo
* npm run build overlay
* npm run build oas

---

Well, in all cases it would be `npm run build-src`, and on second thought it is better to hide the differences inside `package.json` because we plan to move `build.sh` and its parts into a separate package to be reused by all three spec repos.